operator/pkg/ciliumendpointbatch/aggregator_test.go

```diff
@@ -0,0 +1,159 @@
+// Copyright 2021 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build !privileged_tests
+
+package ciliumendpointbatch
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	// CebName  for testing purpose
+	CebName = "ceb-orange-one1"
+)
+
+func TestUpdateAggregator(t *testing.T) {
+	testCases := []struct {
+		name    string
+		cebName string
+		reqs    []CebEventType
+		res     CebEventType
+	}{
+		{
+			name:    "Create, Update and Delete a CEB, this should converge to CebDelete",
```

Comment:
```suggestion
			name:    "Create, Update and Delete a CEB. They should be squashed to a CebDelete",
```
And the rest of them